com.unity.shadergraph/Editor/Generation/Descriptors/PassDescriptor.cs

```diff
@@ -30,6 +30,10 @@ internal struct PassDescriptor
         public AdditionalCommandCollection additionalCommands;
         public CustomInterpSubGen.Collection customInterpolators;
 
+        // Properties
+        public bool useTessellation;
+        public bool useRaytracing;
```

Comment:
Could we hack these in via the Targets interface instead? Ie. allowing passes to be composed with respect to the active context.

Reply:
Not sure to understand what you mean? I am open to suggestion.
If I get it you want it to be contain fully in RenderPipeline ? Note that those parameters really need to be per pass (i.e in a list of pass we can have a mix of tessellatoin, raytracing and regular)

Reply:
Preferably-- perhaps some static branches in the targets could handle this instead via the fields system?

Reply:
Oh-- I mean, it's possible for us to have an entry point where the target asks for the pass information providing active context information. So the branching on which pass information to include can be handled by the target instead of embedding it into the static representation of pass structs we have now.

Reply:
ok figured out what I will do. Preparing change

Reply:
Change done and commited